packages/gatsby-transformer-sharp/src/gatsby-node.js

```diff
@@ -2,6 +2,12 @@ const {
   onCreateNode,
   unstable_shouldOnCreateNode,
 } = require(`./on-node-create`)
+exports.onPreInit = (context, options) => {
+  const { reporter } = context
+  if (reporter.setErrorMap) {
+    reporter.setErrorMap(ERROR_MAP)
+  }
+}
```

Comment:
[nitpick]
We usually do it like following:

```suggestion
exports.onPreInit = ({ reporter }) => {
  if (reporter.setErrorMap) {
    reporter.setErrorMap(ERROR_MAP)
  }
}
```

Reply:
@wardpeet Good to know, thanks! :) I'll use this convention going forward :+1: